docs/docs/02-concepts/03-platforms.md

```diff
@@ -38,6 +38,16 @@ The order in which platforms are evaluated is important.
 
 The first platform in the list is the primary platform, it is responsible for providing the Wing compiler with the base App that will be used to determine how resources are created, as well it will also lay the ground work for what target the rest of the platforms will need to be compatible with.
 
+#### Using Implicit Platforms
+
+Additionally, you can use naming conventions to implicitly define platforms that should be used. These platform files can be located in the root of your project or in a library that your project uses. The naming convention is as follows:
+```sh
+wplatform.js
+*.wplatform.js
+```
+
+For example, if you have a file named `custom.wplatform.js` in the root of your project, it will automatically be added to the list of platforms to be used when compiling your application. Its also important to note that implicit platforms are always loaded after the platforms specified in the `--platform` option.
+
```

Comment:
Maybe add an example to show why this is useful. The original issue has some use case I think.

Reply:
The description is this pull request has some nice documentation